project/src/main/java/com/max/grpc/project/EchoClient.java

```diff
@@ -0,0 +1,45 @@
+
+package com.max.grpc.project;
+
+import com.max.grpc.protos.EchoEntity;
+import com.max.grpc.protos.EchoServiceGrpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EchoClient {

```

Comment:
Client and Server should be separate services. You have to be able to start them even on different servers and they shouldn't know about each other

Reply:
Fixed, maybe